src/OrchardCore/OrchardCore.Data.Abstractions/IDataMigrationManager.cs

```diff
@@ -22,13 +23,14 @@ public interface IDataMigrationManager
         /// Updates the database to the latest version for the specified feature.
         /// </summary>
         /// <param name="feature">The feature to be uninstalled.</param>
-        Task UpdateAsync(string feature);
+        [Obsolete("This method has been deprectated please use UpdateAsync(string[] features) instead.")]
+        Task UpdateAsync(string feature) => UpdateAsync([feature]);
 
         /// <summary>
-        /// Updates the database to the latest version for the specified features.
+        /// Updates the database to the latest version for the specified feature(s).
         /// </summary>
-        /// <param name="features">The features to be updated.</param>
-        Task UpdateAsync(IEnumerable<string> features);
+        /// <param name="features">The feature(s) to be updated.</param>
+        Task UpdateAsync(params string[] features);
```

Comment:
FYI this is a breaking change, but it's not something that anybody should actually notice.